neo/public/javascripts/timeseries/controllers.js

```diff
@@ -39,6 +40,16 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
         }
       }
     );
+
+    console.log("initial width: " + $scope.responsiveWidth);
+    angular.element($window).bind('resize', function() {
+      $scope.responsiveWidth = $window.innerWidth;
+      // manuall $digest required as resize event
+      // is outside of angular
+      $scope.$digest();
+      console.log("width changed: " + $scope.responsiveWidth);

```

Comment:
Hi Jianfeng, this part of code was intended for making the timeseries responsive to resizing event. I think making the width responsive to the initial window size is not good enough. Let me work on making it responsive to resizing event.